components/trial_benefits_modal/trial_benefits_modal.tsx

```diff
@@ -123,10 +124,16 @@ const TrialBenefitsModal = ({
         },
     ], []);
 
+    let trialJustStartedTitle = formatMessage({id: 'trial_benefits.modal.trialStartTitle', defaultMessage: 'Your trial has started! Explore the benefits of Enterprise'});
+
+    if (isCloud) {
+        trialJustStartedTitle = formatMessage({id: 'trial_benefits.modal.trialStartTitleCloud', defaultMessage: 'Your trial has started!'});
+    }
```

Comment:
Why this difference in text for Cloud? It seems both would apply - if there was a change in the design I think the intention is to apply to both. Thoughts @annelieseklein ?

Reply:
this only applies for cloud because of the license update delay, afaik that doesn't happen for the self hosted, meaning that the call to action and title text description should remain the same for self hosted. Am I right @annelieseklein ?